core/src/main/java/org/vertexium/FindPathOptions.java

```diff
@@ -17,6 +18,20 @@ public FindPathOptions(String sourceVertexId, String destVertexId, int maxHops)
         this.sourceVertexId = sourceVertexId;
         this.destVertexId = destVertexId;
         this.maxHops = maxHops;
+        this.getAnyPath = false;
+    }
+
+    /**
+     * @param sourceVertexId The source vertex id to start the search from.
+     * @param destVertexId   The destination vertex id to get to.
+     * @param maxHops        The maximum number of hops to make before giving up.
+     * @param getAnyPath     Return true if a path is found
```

Comment:
Maybe the description of this param should be something more like `return as soon as the first path is found`